app/assets/javascripts/discourse/app/controllers/about.js

```diff
@@ -9,6 +9,14 @@ export default Controller.extend({
     "siteSettings.experimental_rename_faq_to_guidelines"
   ),
 
+  @discourseComputed("siteSettings.experimental_redesigned_about_page_groups")
+  displayV2Design(groups) {
+    const allowedGroupIds = groups.split("|").map((id) => parseInt(id, 10));
+    return this.currentUser?.groups.any((group) =>
```

Comment:
We can't do these group checks on the client, since not all the user's group memberships are sent down to the client at all times. We have to do this on the current user serializer, e.g.

https://github.com/discourse/discourse/blob/e014c907c888ed395332a0aedfc21ebb3003f9cf/app/serializers/current_user_serializer.rb#L137-L139

Reply:
Ah, yes, excellent point. Done, thanks!